src/tests/navBar.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import NavBar from '../components/NavBar';
+
+test('renders NavBar correctly', () => {
+  // Render the NavBar component to a JSON tree
+  const tree = renderer.create(<NavBar />).toJSON();
+
+  // Use Jest's snapshot testing to compare the rendered output to the saved snapshot
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] You only added a screenshot test for the navbar component, As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=the%20home%20page.-,Testing%20requirements,-Create%20unit%20tests), please add unit tests for several components of the app and some pure functions in Redux code and integration tests using React Testing Library by mocking data fetching and Redux Store connection.